src/public/providers.ts

```diff
@@ -97,7 +102,9 @@ class ProviderElement {
         providerKeysContainer
       )
     })
-    provider.domains.map(domain => {
+    const isAuthorized = !provider.domains.code ? true : false
+    const providerDomains = isAuthorized ? provider.domains : []
```

Comment:
why do we need this? `/api/admin/providers` should always be be returning an array of domains. If its not, then the fix must go in at the api endpoint to return `[]` if domains does not exist.